os/net/lwip/src/api/sockets.c

```diff
@@ -299,13 +299,11 @@ struct lwip_socket_multicast_pair {
 
 struct lwip_socket_multicast_pair socket_ipv4_multicast_memberships[LWIP_SOCKET_MAX_MEMBERSHIPS];
 
-static int lwip_socket_register_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr);
-static void lwip_socket_unregister_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr);
-static void lwip_socket_drop_registered_memberships(int s);
+static int lwip_socket_register_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr, pid_t);
+static void lwip_socket_unregister_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr, pid_t pid);
+static void lwip_socket_drop_registered_memberships(struct lwip_sock *);
```

Comment:
You should add the argument name.
`struct lwip_sock *` is not allowed.

Reply:
done